t/dualvar.t

```diff
@@ -131,3 +131,28 @@ SKIP: {
   ok(isdual($ary[2]), 'Is a dualvar');
 }
 
+ok !eval { dualvar() }, "arg count gets checked";
+ok !eval { dualvar(2, "a", "meep") }, "arg count gets checked";
+
+{
+  my $warning;
+  local $SIG{__WARN__} = sub { $warning = shift };
+
+  my $var = dualvar(undef, undef);
+  like($warning, qr/Use of uninitialized value in subroutine entry/, 'undef arg warning');
```

Comment:
Aren't these warnings localised to different languages? Or am I being confused about `$!` ?